e2e/app/element/xpath-defaults.ts

```diff
@@ -14,56 +14,47 @@ export function xPathOptionToXpath(xOpts: XPathOptions, container?: Container):
     if (iconShape === undefined) {
       throw new Error(`Incorrect XPathOptions configuration for Icon: set "iconShape" value.`);
     }
-  } else {
-    if (name === undefined && normalizeSpace === undefined && containsText === undefined) {
-      throw new Error(`Incorrect XPathOptions configuration for label name: 
-      Cannot find a text parameter: set "name", "containsText" or "normalizeSpace" value.`);
-    }
   }
```

Comment:
Removing this check because the Add Criteria dialog has input fields which don't have labels. 🤷‍♂️
Minor tweaking for xpath expressions to accommodate new type of UI fields in Add Criteria dialog.